packages/astro/test/ssr-prerender.test.js

```diff
@@ -61,7 +66,10 @@ describe('SSR: prerender', () => {
 	});
 });
 
-describe('Integrations can hook into the prerendering decision', () => {
+// NOTE: This test doesn't make sense as it relies on the fact that on the client build,
+// you can change the prerender state of pages from the SSR build, however, the client build
+// is not always guaranteed to run. We should have an 
+describe.skip('Integrations can hook into the prerendering decision', () => {
```

Comment:
Maybe @Fryuni is using this in inox-tools?

Reply:
I'm not using it, but Arsh is on `astro-prerender-patterns`:
https://github.com/lilnasy/gratelets/blob/cbb941cb4688f22b713f2e0adec1452b8ec7c106/packages/prerender-patterns/integration.ts#L53-L69

Reply:
In Arsh's implementation, I think it should still work as it's editing the module meta in Vite instead of the `prerender` from Astro, the latter is harder to make it work.